src/memo/memo.ml

```diff
@@ -500,6 +500,25 @@ module Sample_attempt = struct
     | Running { result; _ } -> Once.force result.compute
 end
 
+(* Checking dependencies of a node can lead to one of these outcomes:
+
+   - [Unchanged]: all the dependencies of the current node are up to date and we
+   can therefore skip recomputing the node and can reuse the value computed in
+   the previous run.
+
+   - [Changed]: one of the dependencies has changed since the previous run and
+   the current node should therefore be recomputed.
+
+   - [Cycle_error _]: one of the dependencies leads to a dependency cycle. In
```

Comment:
The doc says `Cycle_error` but the code says `Cancelled`

Reply:
Thanks, fixed!